<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{

background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.994318, stop:0 rgba(119, 203, 239, 255), stop:1 rgba(236, 211, 69, 255))}
</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>681</width>
      <height>481</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 20px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="StartNewGameButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px;
background: #ECD345;
padding: 20px;
width: 200px;
height: 150px;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>НАЧАТЬ ИГРУ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 10px; 
border: 1px solid red;
 border-color: rgb(17, 208, 186);
 color:rgb(67, 65, 49); 
 background-color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
padding-left: 10px;
padding-right: 10 px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 35pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)</string>
     </property>
     <property name="text">
      <string>НОВАЯ ИГРА</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight:bold</string>
     </property>
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LevelComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>205</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 10px; 
border: 1px solid red;
 border-color: rgb(17, 208, 186);
 color:rgb(67, 65, 49); 
 background-color: rgb(255, 255, 255);
padding-left: 10px;

font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Легкий</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight:bold</string>
     </property>
     <property name="text">
      <string>Уровень сложности:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FormComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>285</y>
       <width>300</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 10px; 
border: 1px solid red;
 border-color: rgb(17, 208, 186);
 color:rgb(67, 65, 49); 
 background-color: rgb(255, 255, 255);
padding-left: 10px;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Квадрат</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight:bold</string>
     </property>
     <property name="text">
      <string>Форма:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HeightLabel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>285</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 10px; 
border: 1px solid red;
 border-color: rgb(17, 208, 186);
 color:rgb(67, 65, 49); 
 background-color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
padding-left: 10px;
padding-right: 10 px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>260</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight:bold</string>
     </property>
     <property name="text">
      <string>Размеры:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>297</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight: bold
</string>
     </property>
     <property name="text">
      <string>Выс:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="WidthLabel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>285</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 10px; 
border: 1px solid red;
 border-color: rgb(17, 208, 186);
 color:rgb(67, 65, 49); 
 background-color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
padding-left: 10px;
padding-right: 10 px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>535</x>
       <y>297</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
font-weight: bold
</string>
     </property>
     <property name="text">
      <string>Шир:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ErrorSizeLabel">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>330</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(56, 67, 116)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>400</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px;
background: #ECD345;
padding: 20px;
width: 200px;
height: 150px;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>НАЗАД</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
